lib/parse-args.js

```diff
@@ -0,0 +1,58 @@
+'use strict'
+
+const yargs = require('yargs')
```

Comment:
While I like `yargs`, my only concern is I would like to keep this package to a minimum number of dependencies. I really liked only having one dependency. Maybe if we could just search for the flags ourselves that might be better. 

I am a huge fan of npm and packages and generally making our lives easier, but since I like this util being minimal and a core util, I would like to keep its dependencies very low.

I will comment on the below code after I see what it looks like with the `yargs` package removed.